src/views/main-view.js

```diff
@@ -1,35 +1,15 @@
 import Cycle from '@cycle/core';
-import CycleWeb from '@cycle/web';
+import {h} from '@cycle/web';
 
-import getTHead from './thead';
-import getTBody from './tbody';
+import renderTHead from './thead';
+import renderTBody from './tbody';
 
-import {
-  tableHeightBehavior,
-  rowHeightBehavior,
-  columnsBehavior,
-  rowCountBehavior
-} from '../values';
-
-import visibleIndicesStream from '../streams/visible-indices-stream';
-
-var MainView = Cycle.Rx.Observable.combineLatest(
-  tableHeightBehavior,
-  rowHeightBehavior,
-  columnsBehavior,
-  rowCountBehavior,
-  visibleIndicesStream,
-  (
-    tableHeight,
-    rowHeight,
-    columns,
-    rowCount,
-    visibleIndices
-  ) => {
```

Comment:
See how the view doesn't do anymore the combineLatest, because the model did that for us already. And again we export a function, `view`, instead of a var.